api/server/credentials_linux.go

```diff
@@ -234,14 +234,7 @@ func (s *Server) LogAction(w http.ResponseWriter, r *http.Request) error {
 	}
 	message = fmt.Sprintf("{Action=%v, %s}", action, message)
 	switch action {
-	case "info":
-	case "images":
-	case "version":
-	case "json":
-	case "search":
-	case "stats":
-	case "events":
-	case "history":
+	case "history", "info", "images", "version", "json", "search", "stats", "events":
```

Comment:
I don't understand this switch though. what's the point of the fallthrough? not to logrus.Info()?